src/aeonlib/ocs/blanco/instruments.py

```diff
@@ -1,17 +1,32 @@
 # pyright:  reportUnannotatedClassAttribute=false
 # This file is generated automatically and should not be edited by hand.
 
-from typing import Any, Annotated, Literal
+from typing import Any, Annotated, Literal, Optional
 
-from annotated_types import Le
-from pydantic import BaseModel, ConfigDict
+from annotated_types import Le, Ge
+from pydantic import BaseModel, ConfigDict, Field
 from pydantic.types import NonNegativeInt, PositiveInt
 
 from aeonlib.models import TARGET_TYPES
 from aeonlib.ocs.target_models import Constraints
 from aeonlib.ocs.config_models import Roi
 
 
+
+class BlancoNewfirmConfigExtraParams(BaseModel):
+    model_config = ConfigDict(validate_assignment=True, extra='allow')
+    dither_value: Annotated[int, Ge(0), Le(1600)] = 80
+    dither_sequence: Literal[['2x2', '3x3', '4x4', '5-point']] = "2x2"
+    detector_centering: Literal[['none', 'det_1', 'det_2', 'det_3', 'det_4']] = "det_1"
+    dither_sequence_random_offset: Literal[[True, False]] = True
```

Comment:
Isn't this... "Literally" a boolean? ✔️

Reply:
yes, but it's defined this way in the cerberus schema to get us a select dropdown in the UI... The effect in the library should be the same as if it was an Annotated[bool] right?